uint16 pwm_ratio1
uint16 pwm_ratio2
uint8  gear_ratio
uint8  all_wheel_drive
time   time_received
